### ISSUE TITLE
E: Unable to correct problems, you have held broken packages

### ISSUE THREAD

**Issue by keekkenen** [openwrt-xiaomi/awg-openwrt#6]
# apt install -y amneziawg
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 amneziawg-dkms : Depends: dkms (>= 2.1.0.0) but it is not installable

Operating System: Ubuntu 22.04.5 LTS               
Kernel: Linux 5.15.0-125-generic
Architecture: x86-64

**Comment by Gyodzuru** [comment]
Удалось решить проблему?

**Comment by keekkenen** [comment]
> Удалось решить проблему?

нет, я просто перечитал доку на сайте и где-то увидел, что есть другой путь - установить приложение амнезии на любое устройство и с него за 5 минут все настроить
